model_servers/llamacpp_python/tests/requirements.txt

```diff
@@ -0,0 +1,8 @@
+pip==24.0
```

Comment:
Do we care about specific versions?  IE Will this become out of date over time?  Or do we plan on using renovate?

Reply:
I think its better to have all deps version locked, it will ensure that demos will be more robust. Ideally we also find a way to manage dependency updates like renovate or dependabot but for now I think this is fine

Reply:
renovate is planned to be onboarded